pyanaconda/progress.py

```diff
@@ -49,8 +50,11 @@ def progress_report(message):
     yield
     progress_step(message)
 
-def progress_message(message):
-    progressQ.send_message(message)
+def progress_message(message, xlate=True):
```

Comment:
I'd rather see all the places to be update to pass an untranslated string because we really want the untranslated string to be logged.